tests/memory/short_term_memory_test.py

```diff
@@ -26,7 +26,7 @@ def short_term_memory():
     return ShortTermMemory(crew=Crew(agents=[agent], tasks=[task]))
 
 
-@pytest.mark.vcr(filter_headers=["authorization"])
+# @pytest.mark.vcr(filter_headers=["authorization"])
```

Comment:
_Suggestion_: I think you did this to run VCR create a new cassete and put it back again is that it?

Reply:
good catch. yes